app/models/user_acceptable_verified_credential.rb

```diff
@@ -8,4 +8,8 @@ class UserAcceptableVerifiedCredential < ApplicationRecord
   scope :without_avc,     -> { where(acceptable_verified_credential_at: nil) }
   scope :without_ivc,     -> { where(idme_verified_credential_at: nil) }
   scope :without_avc_ivc, -> { where(acceptable_verified_credential_at: nil, idme_verified_credential_at: nil) }
+
+  def verified_credential?
+    acceptable_verified_credential_at || idme_verified_credential_at
+  end
```

Comment:
I don't think we want helper methods like this in the model. Could you move this logic to your service class?